resources/css/_tables.css

```diff
@@ -226,6 +226,10 @@ body.table-compact-until-lg .arrow-direction {
     @apply bg-theme-secondary-200 dark:bg-black !important;
 }
 
+.table-encapsulated table > tbody > tr > td.hoverable-cell {
+    @apply align-top;
```

Comment:
although this should work, the current implementation has a 2px difference between the `amount` column (47px tall) and the other column contents (45px tall). So although it looks centered, the entries are not by 2px. May want to discuss with Sam/Oleg if that's ok
<img width="333" alt="image" src="https://github.com/ArdentHQ/arkscan/assets/35610748/0580965e-ff94-4d1f-a225-c47826d97ae1">
<img width="326" alt="image" src="https://github.com/ArdentHQ/arkscan/assets/35610748/23d25be6-c94d-4330-83b4-bbd6db39f167">
can see in the screenshots that the amount padding gets closer to the bottom dashed line